validation-test/SIL/parse_stdlib.sil

```diff
@@ -3,3 +3,12 @@
 // RUN: %target-sil-opt -assume-parsing-unqualified-ownership-sil -enable-sil-verify-all=true %t.sil > /dev/null
 // REQUIRES: long_test
 // REQUIRES: nonexecutable_test
+// https://github.com/apple/swift/pull/12430 makes an assumption that
+// the first parameter of a method is the self type that can be used to
+// look up conformances.
+// That's generally reasonable, but not for IUOs where we lower the
+// types to plain Optionals. When IUOs have no extensions containing
+// methods, we're fine, but in https://github.com/apple/swift/pull/13976
+// I am reintroducing extensions because removing them broke things
+// (bridging to ObjectiveC, for one).
+// REQUIRES: disabled
```

Comment:
If we remove the two extensions I flagged above, we could leave this test enabled for Linux.

Reply:
Yes, true. I'll look at that. I was just hoping to revert and move forward, but if we're confident we won't hit other issues from these others being removed then we should do that.